library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -1045,7 +1045,6 @@ void libspdm_reset_message_buffer_via_request_code(void *context, void *session_
         }
         break;
     case SPDM_GET_DIGESTS:
-    case SPDM_GET_CERTIFICATE:
         if (spdm_context->connection_info.connection_state ==
             LIBSPDM_CONNECTION_STATE_AUTHENTICATED) {
             libspdm_reset_message_b(spdm_context);
```

Comment:
I am confused. How does this work?

Reply:
When second GetDigest, the connection_state is LIBSPDM_CONNECTION_STATE_AUTHENTICATED.